client/src/components/Login/Login.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import {Button, Modal} from 'react-bootstrap'
+
+function Log(props){
+  return(
+    <div>
+          <Modal id = "login" show = {props.show} onHide = {props.onRequestClose}>
```

Comment:
1. The indention looks inconsistent compared to the component above. 
2. You might want to also get rid of the extra spacing here.
   `<Modal id="login" show={props.show} onHide={props.onRequestClose}>`